xpm_ruby.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.summary       = "Ruby gem for accessing the XPM API"
   spec.homepage      = "https://github.com/ignitionapp/xpm_ruby"
   spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.6.0")
```

Comment:
How have we merged things until now, when there are many rubocop offences in the current codebase?